opennms-doc/guide-development/src/asciidoc/text/topology/graphml.adoc

```diff
@@ -125,10 +125,27 @@ Instead if a custom icon should be used, each node must contain a `iconKey` prop
 [[gd-topology-graphml-vertex-status-provider]]
 ==== Vertex Status Provider
 
-The _Vertex Status Provider_ calculates the status of the Vertex based on the "worst" unacknowledged alarm associated with the Vertex's node.
-This means in order to have a status calculated a (OpenNMS) Node must be associated with the Vertex.
-This can be achieved by setting the GraphML attribute `nodeID` on the GraphML Node accordingly.
+The _Vertex Status Provider_ calculates the status of the Vertex.
+There are three implementations available which can be configured for each graph: `default`, `script` and `propagate`.
```

Comment:
Can we replace `three` with `multiple`, therefore we don't have to patch this after we change dthe number of implementations because we forget to update the number (-: